Gamma_Analysis.py

```diff
@@ -418,12 +420,15 @@ def check_spectra(samples, background, reference):
                             error_spectrum)
 
 
-def main():
-    background = SPEFile.SPEFile("USS_Independence_Background.Spe")
+def main(background_filename=BACKGROUND, reference_filename=REFERENCE, 
+         file_list=None):
+    
+    background = SPEFile.SPEFile(background_filename)
     background.read()
-    reference = SPEFile.SPEFile("UCB018_Soil_Sample010_2.Spe")
+    reference = SPEFile.SPEFile(reference_filename)
     reference.read()
-    sample_measurements, sample_names = acquire_files()
+    if file_list is None:
+        sample_measurements, sample_names = acquire_files()
```

Comment:
I have written the method. Right now it can pick out the spectra in file_list and display the ROI plots. However no tables can be written at the moment. "Metadata in RadWatch_Samples.csv doesn't match the SPE files in this directory! Not making output CSV's" is called since the make_table method in gamma analysis creates a length of masses from the RadWatch_Samples.csv (which contains all the samples, even the ones that are not in the files_list). I'm going to start restructuring how gamma_analysis reads csv now since the file_list input argument won't be able to work until this is done.